packages/mongodb-rag-ingest/src/sources/mongodb-university/makeUniversityPages.ts

```diff
@@ -181,3 +202,26 @@ export function convertVideoTranscriptFromSrtToTxt(transcript: string): string {
     .join(" ")
     .trim();
 }
+
+/**
+  Helper function to create Markdown content for MongoDB University Learning Paths and Courses
+  based on titles, duration, and descriptions.
+ */
+export function generateMarkdown({
```

Comment:
```suggestion
export function generateContentDescriptionMarkdown({
```

feel free to play w/ the name, but let's make it more descriptive